### ISSUE TITLE
twisted.internet.udp.Port.write() docstring should mention it uses bytes

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8635]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8635|
|Type|enhancement|
|Created|2016-07-20 11:53:44Z|
|Branch|https://github.com/twisted/twisted/tree/8635-rodrigc-udp-docstring|

The docstrings for twisted.internet.udp.Port.write()
and twisted.internet.udp.Port.writeSequence() should
mention that they take bytes parameters, not str parameters

<details><summary>Searchable metadata</summary>

```
trac-id__8635 8635
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8635_rodrigc_udp_docstring 8635-rodrigc-udp-docstring
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1469015624164308 1469015624164308
changetime__1469103451417242 1469103451417242
version__None None
owner__rodrigc rodrigc

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_3"></a>|@rodrigc commented|
|-|-|

See:
https://github.com/twisted/twisted/pull/369

**Comment by twisted-trac** [comment]
|[<img alt="scop's avatar" src="https://avatars.githubusercontent.com/u/109152?s=50" width="50" height="50">](https://github.com/scop)<a name="note_4"></a>|@scop set owner to @rodrigc|
|-|-|

LGTM, please merge

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_5"></a>|GitHub set status to `closed`|
|-|-|

In changeset 127aa16b0a60b7e1b505017ec89c637b4e603fbb
```
#!CommitTicketReference repository="" revision="127aa16b0a60b7e1b505017ec89c637b4e603fbb"
Merge pull request #369 from twisted/8635-rodrigc-udp-docstring

Author: rodrigc
Reviewer: scop
Fixes: #8635

Clarify docstring for twisted.internet.udp.Port.write and twisted.internet.udp.Port.writeSequence
```
